Src/MoneyFox.Windows/CortanaFunctions.cs

```diff
@@ -0,0 +1,226 @@
+            XElement createpaymentlistenfor = new XElement(xds+"ListenFor");
+            XElement createpaymentfeedback = new XElement(xds+"Feedback");
+            createpayment.SetAttributeValue("Name", "create-payment");
+            createpaymentExample.SetValue($"{Strings.CortanaVoiceCommandCreatePaymentExample}");
+           /// createpaymentlistenfor.SetAttributeValue("RequireAppName", "AfterPhrase");
+            createpaymentlistenfor.SetValue($"{Strings.CortanaVoiceCommandCreatePaymentListenFor}");
+            createpaymentfeedback.SetValue($"{Strings.CortanaVoiceCommandCreatePaymentFeedback}");
+            createpayment.Add(createpaymentExample);
+            createpayment.Add(createpaymentlistenfor);
+            createpayment.Add(createpaymentfeedback);
+            createpayment.Add(VoiceCommandService);
+
+          CommandSet.Add(createpayment);
+            /// payment amount section
+            XElement paymentamount = new XElement(xds+"Command");
+            XElement paymentamountExample = new XElement(xds+"Example");
+            XElement paymentamountlistenfor = new XElement(xds+"ListenFor");
+            XElement paymentamountfeedback = new XElement(xds+"Feedback");
+            paymentamount.SetAttributeValue("Name", "payment-amount");
+            paymentamountExample.SetValue($"{Strings.CortanaVoiceCommandPaymentAmountExample}");
+           /// paymentamountlistenfor.SetAttributeValue("RequireAppName", "AfterPhrase");
+            paymentamountlistenfor.SetValue($"{Strings.CortanaVoiceCommandPaymentAmountListenFor}");
+            paymentamountfeedback.SetValue($"{Strings.CortanaVoiceCommandPaymentAmountFeedback}");
+            paymentamount.Add(paymentamountExample);
+            paymentamount.Add(paymentamountlistenfor);
+            paymentamount.Add(paymentamountfeedback);
+            paymentamount.Add(VoiceCommandService);
+            CommandSet.Add(paymentamount);
+            /// payment date section
+            XElement paymentdate = new XElement(xds+"Command");
+            XElement paymentdateExample = new XElement(xds+"Example");
+            XElement paymentdatelistenfor = new XElement(xds+"ListenFor");
+            XElement paymentdatefeedback = new XElement(xds+"Feedback");
+            paymentdate.SetAttributeValue("Name", "payment-date");
+            paymentdateExample.SetValue($"{Strings.CortanaVoiceCommandPaymentDateExample}");
+            ///paymentdatelistenfor.SetAttributeValue("RequireAppName", "AfterPhrase");
+            paymentdatelistenfor.SetValue($"{Strings.CortanaVoiceCommandPaymentDateListenFor}");
+            paymentdatefeedback.SetValue($"{Strings.CortanaVoiceCommandPaymentDateFeedback}");
+            paymentdate.Add(paymentdateExample);
+            paymentdate.Add(paymentdatelistenfor);
+            paymentdate.Add(paymentdatefeedback);
+            paymentdate.Add(VoiceCommandService);
+           CommandSet.Add(paymentdate);
+            ///create dollars phrase topic
```

Comment:
code comments are only 2 dashes. 3 are for documentation on methods, but that's not a valid point for them here :)